src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
```

Comment:
Nitpick: I think it would be better if you grouped the imports (require) at the top, and put this line after the two imports!